indico/modules/events/editing/client/js/management/editable_type/NextEditable.jsx

```diff
@@ -49,13 +51,15 @@ NextEditable.propTypes = {
   eventId: PropTypes.number.isRequired,
   editableType: PropTypes.oneOf(Object.values(EditableType)).isRequired,
   onClose: PropTypes.func.isRequired,
+  management: PropTypes.bool.isRequired,
 };
 
-function NextEditableDisplay({eventId, editableType, onClose, fileTypes}) {
+function NextEditableDisplay({eventId, editableType, onClose, fileTypes, management}) {
```

Comment:
Perhaps later, to avoid stalling this PR, we could merge `NextEditable` with `NextEditableDisplay` since the latter isn't used individually. I don't think there's any reason to keep them separate.

Reply:
The `NextEditableDisplay` component has a `useEffect` which loads the actual editables - having a separate component preload all the things that never change has the advantage that you can just return null or the loader there, and don't have to run the hooks of the main component all the time.

Ideally we would just load all the static data needed on the various pages on a higher level and provide it via context though... But I don't think it's that important here since there's not much overhead in loading it here.